lighthouse-core/report/html/renderer/details-renderer.js

```diff
@@ -75,8 +75,10 @@ class DetailsRenderer {
    */
   _renderBytes(details) {
     // TODO: handle displayUnit once we have something other than 'kb'
-    const value = Util.i18n.formatBytesToKB(details.value, details.granularity);
-    return this._renderText(value);
+    const value = Util.i18n.formatBytesToKiB(details.value, details.granularity);
+    const text = this._renderText(value);
+    text.setAttribute('title', Util.i18n.formatNumber(details.value));
```

Comment:
wait, why do this? You mentioned doing this in the PR description but didn't say why :) I don't have any objection just wondering why it's beneficial.

Reply:
I appreciate when tools do this with labels I'm unfamiliar with. If I'm unsure if KiB means 1000 or 1024, a quick hover to see the real value relative to the elided value tells me right away.

also it's nice to just be able to see the real size

Reply:
> also it's nice to just be able to see the real size

note it's not always a real size :) (estimated opportunities and sometimes already rounded to various degrees).

Should it `+= ' bytes'` then?

Reply:
alrighty I made `formatBytes`. @exterkamp says this should be i18n'd but the other value rendering functions here don't do that yet so I'm gonna defer it here too.